components/Usercard.js

```diff
@@ -0,0 +1,26 @@
+export default function Usercard() {
+  const user = {
```

Comment:
Ich würde die User Daten wahrscheinlich nicht in der Komponente speichern. Vielleicht packst du sie erstmal in die index ?

Reply:
Ja die kommen im nächsten step aus einer Form-Component da werde ich sie liften👍